src/ansi.c

```diff
@@ -218,4 +207,27 @@ wattrset_by_ansi_status(struct view *view, struct ansi_status* cur_ansi_status)
 	wattr_set(view->win, cur_ansi_status->attr, id, NULL);
 }
 
+short
+convert_ansi_into_256_color(char *save_ptr) {
```

Comment:
I believe `save_ptr` needs to be a `char**` here because `strtok_r` modifies the value of the pointer.  As it is currently written, when the value of `save_ptr` gets modified, the changed value isn't passed back into the calling function.

I discovered this problem when it tripped over the following ANSI code: `\033[38;5;8m`.  The "38" triggered us to enter this function, the "5" was handled here, but upon returning, the "5" was parsed again and it caused all my code to start blinking.  It was pretty funny actually ;)

I tested this change on my side and things started behaving correctly.

Reply:
You are correct, sorry for my mistake. Fixed here [5093bb1](https://github.com/jonas/tig/pull/1140/commits/5093bb1f89ec4309956fd4101e46aa64cce485a8).

This could be an excuse, but the reason why I didn't notice is that iTerm and VSCode-Terminal doesn't support BLINK.... And Macbook-builtin Terminal does support. Now I confirmed the bug and fixed. 
https://unix.stackexchange.com/questions/151673/attrona-blink-using-ncurses-does-nothing 

Thank you very much for catching again.